src/site/includes/metatags.liquid

```diff
@@ -31,8 +31,11 @@
 <meta content="{{ tag }}" property="article:tag">
   {% endfor %}
 {% endif %}
+{% if keywords %}
+<meta name ="keywords" content="{{keywords}}">
+{% endif %}
 {% if title === 'Home' %}
 <title>VA.gov</title>
 {% else %}
-<title>{{ title }}: VA.gov</title>
+<title>{{ title }} | Veterans Affairs</title>
```

Comment:
I think is looks good 👍 Only thing is that I think we should also check on the title of the homepage. I'm curious if we want to leave it as `VA.gov`, or if we should update it to `Veterans Affairs`.

Reply:
Thanks, Will do.

Also making edits to the e2e tests too since I many of them were looking for page titles w/ `: VA.gov` at the end.

Reply:
Ah yes, good catch @aubreyarcangel!